src/NServiceBus.RabbitMQ.AcceptanceTests/ConfigureEndpointRabbitMQTransport.cs

```diff
@@ -20,12 +20,36 @@ class ConfigureScenariosForRabbitMQTransport : IConfigureSupportedScenariosForTe
 
 class ConfigureEndpointRabbitMQTransport : IConfigureEndpointTestExecution
 {
-    string connectionString;
+    DbConnectionStringBuilder connectionStringBuilder;
 
     public Task Configure(string endpointName, EndpointConfiguration configuration, RunSettings settings)
     {
-        connectionString = settings.Get<string>("Transport.ConnectionString");
-        configuration.UseTransport<RabbitMQTransport>().ConnectionString(connectionString);
+        connectionStringBuilder = new DbConnectionStringBuilder
+        {
+            ConnectionString = Environment.GetEnvironmentVariable("RabbitMQTransport.ConnectionString"),
+        };
+
+        if (!connectionStringBuilder.ContainsKey("username"))
+        {
+            connectionStringBuilder.Add("username", "guest");
+        }
+
+        if (!connectionStringBuilder.ContainsKey("password"))
+        {
+            connectionStringBuilder.Add("password", "guest");
+        }
+
+        if (!connectionStringBuilder.ContainsKey("virtualhost"))
+        {
+            connectionStringBuilder.Add("virtualhost", "nsb-rabbitmq-test");
+        }
+
+        if (!connectionStringBuilder.ContainsKey("host"))
+        {
+            connectionStringBuilder.Add("host", "localhost");
+        }
```

Comment:
If all these `if` blocks are distasteful, I can extract a method and collapse these down to four lines. E.g.

```C#
ApplyDefault(connectionStringBuilder, "username", "guest");
...
```